torch/csrc/stable/tensor.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
+
+// TODO ASAP: THIS FILE SHOULD BE HEADER ONLY BUT ISN'T ENFORCED:
+// I only need it for AOTI_TORCH_ERROR_CODE_CHECK, see #154908
+#include <torch/csrc/inductor/aoti_runtime/utils.h>
+
+#include <torch/csrc/inductor/aoti_torch/c/shim.h>
+
+namespace torch::stable {
+
+using DeviceIndex =
+    int8_t; // this is from c10/core/Device.h and can be header only
+
+inline void delete_tensor_object(AtenTensorHandle ath) {
+  AOTI_TORCH_ERROR_CODE_CHECK(aoti_torch_delete_tensor_object(ath));
+}
+
+// The torch::stable::Tensor class is a highlevel C++ header-only wrapper around
```

Comment:
```suggestion
// The torch::stable::Tensor class is a highlevel C++ wrapper around
```
To reduce confusion with standalone